src/Service/HttpTranslator.php

```diff
@@ -20,7 +20,7 @@
 /**
  * @author Tobias Nyholm <tobias.nyholm@gmail.com>
  */
-abstract class HttpTranslator
+abstract class HttpTranslator extends AbstractTranslator
```

Comment:
Your new AbstractTranslator is only used on the abstract HttpTranslator, so maybe it could be part of HttpTranslator directly?